### DISCUSSION TITLE
Nano 33 IoT (SAMD21) Support on TinyUSB for Arduino

### DISCUSSION THREAD

**Discussion by nawozef** [hathach/tinyusb#919] [Ideas]
Hello Dear team of TinyUSB, is possible for you to add a Nano 33 IoT (SAMD21) Support? I saw the chip listed on the Support but not on Arduino and i have trying to make a WebUSB with this Arduino (On WebUSB is listed with compatibility but I tried to run natively for more than 3 months without sucess). I like it because it's super stable with different voltage levels and the capability internal achieve the spectations.  Thanks a lot

**Comment by hathach** [comment]
Do you really understand my answer in the other topic ?? If you want to run tinyusb on this board with Arduino, you should go to Arduino forum and ask them to add support for it. Else, try to pick of the samd board from Adafruit. Please don't open another discussion on the same issue. If you still have question, please continue on the old one.
